.agents/skills/supply-chain-audit/references/detection-patterns.md

```
@@ -0,0 +1,587 @@
+1. Observe the style and content of legitimate commits
+2. Create a malicious commit with a message copied from an earlier commit
+3. The commit appears routine at first glance
+
+A commit by "Author B" with the same message as an earlier commit by
+"Author A" warrants investigation.
+
+### Jaccard similarity
+
+We use Jaccard similarity on word tokens:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)